tests/bpf2c_test_wrapper/dllmain.cpp

```diff
@@ -85,15 +86,14 @@ get_metadata_table(const char* name)
     FIND_METADATA_ENTRY(name, printk);
     FIND_METADATA_ENTRY(name, printk_legacy);
     FIND_METADATA_ENTRY(name, reflect_packet);
+    FIND_METADATA_ENTRY(name, sockops);
     FIND_METADATA_ENTRY(name, tail_call);
     FIND_METADATA_ENTRY(name, tail_call_bad);
     FIND_METADATA_ENTRY(name, tail_call_multiple);
+    FIND_METADATA_ENTRY(name, test_sample_ebpf);
     FIND_METADATA_ENTRY(name, test_utility_helpers);
 #if defined(_DEBUG)
-    FIND_METADATA_ENTRY(name, cgroup_sock_addr);
-    FIND_METADATA_ENTRY(name, sockops);
     FIND_METADATA_ENTRY(name, tail_call_map);
```

Comment:
Is program verification failing for this test?

Reply:
```
Verification failed for xdp_prog with error Verification failed
 Report:

exit: Map cache entry for original map fd 18446744073709551615 not found.
```

Reply:
https://github.com/microsoft/ebpf-for-windows/issues/973